tests/test_adjointJ.py

```diff
@@ -12,11 +13,12 @@
 @pytest.mark.parametrize('space_order', [4, 8, 12])
 @pytest.mark.parametrize('shape', [(60, 70), (40, 50, 30)])
 def test_acousticJ(shape, space_order):
+    clear_cache()
```

Comment:
I think I wrote it somewhere but I cannot find it. ``clear_cache`` should be called only at the very end of those tests that are going to allocate lots of memory, not at the beginning of some (more or less) random tests